ci/test-checks.sh

```diff
@@ -16,14 +15,14 @@ echo --- build environment
   rustup run "$rust_stable" rustc --version --verbose
   rustup run "$rust_nightly" rustc --version --verbose
 
-  cargo +"$rust_stable" --version --verbose
-  cargo +"$rust_nightly" --version --verbose
+  "${cargo}" stable --version --verbose
```

Comment:
Applicable to the entire PR: I think I prefer `"$cargo"` to `"${cargo}"`.   Less hard-to-read #$%@$!#!}#{)(^$%^$% syntax to obscure intent.

Ironically I was once a militant `${purist}`...

Reply:
Ha!  Can do.  I prefer it since I inevitably want to manipulate the string or add some ambiguating character to the end.  Just a preference though :wink: